_action_files/check_js.sh

```diff
@@ -2,17 +2,17 @@
 # The purpose of this script is to check parity between official hosted  third party js libraries, and alternative CDNs used on this site.
 
 function compare {
-            printf "=================\ncomparing:\n$1 vs. $2\n"
-            wget "$1" -O f1 &> /dev/null
-            wget "$2" -O f2 &> /dev/null
-            if ! cmp f1 f2; 
-                then
-                    printf "Files are NOT the same!\n"
-                    exit 1;
-                else
-                    printf "Files are the same.\n"
-            fi 
-            }  
+    printf "=================\ncomparing:\n%s vs. %s\n" "$1" "$2"
+    wget "$1" -O f1 &> /dev/null
+    wget "$2" -O f2 &> /dev/null
+    if ! cmp f1 f2;
+    then
+        printf "Files are NOT the same!\n"
+        exit 1;
+    else
+        printf "Files are the same.\n"
+    fi
+}
 
```

Comment:
I'm not sure this change is necessary at all

Reply:
Reverted formatting changes.
Line5 actually solves [SC2059](https://github.com/koalaman/shellcheck/wiki/SC2059); this is why this change is still there. For example, the new code could handle a `%` in a link you provide for comparison, where the code right now cannot.